_data/simple-icons.json

```diff
@@ -5100,6 +5100,18 @@
             "hex": "1FB141",
             "source": "https://icons8.com/"
         },
+        {
+            "title": "ICP",
```

Comment:
There are a lot of products and organizations on the internet called ICP. I recommend using `Internet Computer Protocol` instead.

Reply:
Renamed to `Internet Computer` (for consistency with [internetcomputer.org](https://internetcomputer.org/)). 

Thanks for taking a look at this PR!

Reply:
@simple-icons/icon-review Any thoughts on this icon name and its aliases? Can we use `ICP` as an alias with this icon?